extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -271,9 +271,6 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		}
 
 		extend(params, decodeLegacyDartParams(targeting.wikiCustomKeyValues));
-		if (lookups) {
```

Comment:
Yup, it's removed here:
https://github.com/Wikia/app/pull/8492/files#diff-21e971af0f2286c20f3673208921310aL97